src/scripts/main.js

```diff
@@ -1,3 +1,10 @@
 'use strict';
 
 // write your code here
+const trs = document.querySelectorAll('table tr');
+
+for (const tr of trs) {
+  const p = tr.cells[1].cloneNode(true);
+
+  tr.append(p, tr.cells[tr.cells.length - 1]);
```

Comment:
The `append` method is used incorrectly here. The `append` method should take individual nodes or strings as arguments, not a combination of a node and another cell. You should append only the cloned node `p` to the row. If you want to move the last cell, you should handle it separately.